src/js/engagement_view/src/components/graphDisplay/GraphDisplay.tsx

```diff
@@ -6,7 +6,6 @@ import {
     calcLinkDirectionalArrowRelPos,
 } from "./graphVizualization/linkCalcs";
 import { nodeSize } from "./graphVizualization/nodeCalcs";
-import { getLinkLabel } from "./graphLayout/labels";
 import { updateGraph } from "./graphUpdates/updateGraph";
 import { Link, VizNode, VizGraph } from "../../types/CustomTypes";
```

Comment:
Quick question: is a Link here a graph edge? it might be worth renaming these to `edge`  in a future PR; i literalyl thought this was like `<a href=`s

Reply:
Correct. The term "link" comes from force-graph.